deployment/envgen2/EnvGen.cpp

```diff
@@ -712,15 +738,16 @@ void CEnvGen::usage()
   puts("          directory and to set eclwatch's enableSystemUseRewrite to true, provide the following options:");
   puts("             \"-override DropZone,@directory,/mnt/disk1/mydropzone ");
   puts("               -override espsmc,@enableSystemUseRewrite,true\"");
-  puts("   -set_xpath_attrib_value <XPATH> <ATTRIBUTE> <VALUE>: sets or add the xpath attribute and value.");
-  puts("          Example: \"-set_xpath_attrib_value  Software/Topology/Cluster[@name=\"thor\"]/ThorCluster @process thor123\"");
-  puts("   -add_content <XPATH> <content file in XML or JSON format>: adds  content to the xpath.");
-  puts("          Example: \"-add_content  Software/Topology /tmp/new_topology_cluster.xml");
-  puts("   -env_options environment options file. The default is /etc/HPCCSystems/environment.conf.");
-  puts("   -env_rules: a rule file to generate environment. The default is /etc/HPCCSystems/genenvrules.conf.");
-  puts("   -env_rules: a rule file to generate environment. The default is /etc/HPCCSystems/genenvrules.conf.");
+  puts("   -set-xpath-attrib-value <XPATH> <ATTRIBUTE> <VALUE>: sets or add the xpath attribute and value.");
+  puts("          Example: \"-set-xpath-attrib-value  Software/Topology/Cluster[@name=\"thor\"]/ThorCluster @process thor123\"");
+  puts("   -add-content <XPATH> <content file in XML or JSON format>: adds  content to the xpath.");
+  puts("          Example: \"-add-content  Software/Topology /tmp/new_topology_cluster.xml");
+  puts("   -env-options environment options file. The default is /etc/HPCCSystems/environment.conf.");
+  puts("   -env-rules: a rule file to generate environment. The default is /etc/HPCCSystems/genenvrules.conf.");
+  puts("   -env-rules: a rule file to generate environment. The default is /etc/HPCCSystems/genenvrules.conf.");
   puts("   -buildset: the buildset file with full path. The directory also contains all XSD files.");
   puts("              the default is /opt/HPCCSystems/componentfiles/configxml/buildset.xml.");
+  puts("   -cloud <none|aws>: set certain attributes for particular cloud environment. The default is none");
```

Comment:
Looks as if you change the connector character from '_' to '-'. Are there any affected scripts that may use the old form? Perhaps Michael should be consulted?